### PR TITLE
Integration test New-RsFolder

### PR DESCRIPTION
**Pull request by erimaja**
Changes proposed in this pull request:
 - Adding Pester framework for integration testing
 - Adding integration testing for New-RsFolder
 -Adding installation of reporting services for allowing integration test to run

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -13,9 +13,32 @@ skip_commits:
...
+- ps: Install-Module -Name Pester
 
 build: false
 
+before_test:
+- ps: |
+    $ErrorActionPreference = 'Stop';
+    $url = "$($env:downloadUrl)/SQLServer2016-x64-ENU.exe"
+    $urlBox = "$($env:downloadUrl)/SQLServer2016-x64-ENU.box"
+    #arguments sqlserver needs
```
Comment (jtarquino) [inline appveyor.yml]: please add needs for installation

File: appveyor.yml [hunk 2]
```diff
@@ -30,7 +53,18 @@ test_script:
     $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\TestsResults.xml))
     # Fail if thre are issues
     if($results.Count -gt 0){throw "ScriptAnalyzer found $($results.Count) issues"}
-   
+    # Activate verbose on Powershell
```
Comment (jtarquino) [inline appveyor.yml]: I would add an extra space and a comment that mention that the next section is about to perform the integration tests

File: Tests/New-RsFolder.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Describe "New-RsFolder" {
+    Context "Creat Folder with minimun parameters"{
+        $folderName = 'SutFolderMinParameters' + [guid]::NewGuid()
+        New-RsFolder -Path / -FolderName $folderName
+        $folderList = Get-RsFolderContent -RsFolder /
+        $folderCount = ($folderList | where name -eq $folderName).count
```
Comment (jtarquino) [inline Tests/New-RsFolder.Tests.ps1]: nit: Count

**Comment by jtarquino** [comment]
> <?xml version="1.0" encoding="utf-8" standalone="no"?>

This file is generated in very test please remove it from the repo

---
Refers to: TestsResults.xml:1 in f702c38. [](commit_id = f702c38302d3021d344fc2a999351e7f5767db5c, deletion_comment = False)

**Comment by jtarquino** [comment]
Thanks a lot for doing this!!!!!

File: appveyor.yml (hunk 1 above)
Comment (erimaja) [inline appveyor.yml]: resolved

File: appveyor.yml (hunk 2 above)
Comment (erimaja) [inline appveyor.yml]: resolved

File: Tests/New-RsFolder.Tests.ps1 (hunk 3 above)
Comment (erimaja) [inline Tests/New-RsFolder.Tests.ps1]: resolved

**Comment by rgernha** [comment]
Thanks a lot, this is fantastic :) :)

**Pull request merged**